### PR TITLE
modify ini config file as  yaml

### PR DESCRIPTION
**Pull request by xwz123**

### CODE REVIEW

File: conf/app.conf.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+python_bin: /usr/bin/python3
+
+cla_fields_number: 10
+cla_platform_url: https://clasign.osinfra.cn
+
+verification_code_expiry: 300
+api_token_expiry: 3600
+api_token_key: fsfsfsafsfsasaf242342424sdfs;.]{77&&&
+symmetric_encryption_key: key-can-be--16-24-32-bytes-long!
```
Comment (zengchen1024) [inline conf/app.conf.yaml:9]: symmetric_encryption_key is not supported now

File: config/config.go [hunk 2]
```diff
@@ -4,19 +4,26 @@ import (
...
+	return AppConfig.validate()
+}
 
 type appConfig struct {
 	PythonBin                 string        `json:"python_bin" required:"true"`
 	CLAFieldsNumber           int           `json:"cla_fields_number" required:"true"`
 	MaxSizeOfCorpCLAPDF       int           `json:"max_size_of_corp_cla_pdf"`
 	MaxSizeOfOrgSignaturePDF  int           `json:"max_size_of_org_signature_pdf"`
+	MinLengthOfPassword       int           `json:"min_length_of_password"`
+	MaxLengthOfPassword       int           `json:"max_length_of_password"`
```
Comment (zengchen1024) [inline config/config.go:26]: don't import unused configuration items.

**Comment by zengchen1024** [comment]
modify the config of deploy

File: deploy/app.conf.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+code_platforms: ./conf/platforms/code_platforms.yaml
+email_platforms: ./conf/platforms/email.yaml
+
+employee_managers_number: 5
+
+max_size_of_corp_cla_pdf: 5242880
+max_size_of_org_signature_pdfa: 204800
+
+cla_platform_url: "${CLA_PLATFORM_URL}"
+
```
Comment (zengchen1024) [inline deploy/app.conf.yaml:21]: missing restricted_corp_email_suffix

File: deploy/app.conf.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+mongodb:
+  mongodb_conn: "${MONGODB_CONNECTION}"
+  mongodb_db: "${MONGODB_NAME}"
+  link_collection: links
+  corp_pdf_collection: corp_pdfs
```
Comment (zengchen1024) [inline deploy/app.conf.yaml:26]: missing cla_pdf_collection

**Review by zengchen1024 — CHANGES_REQUESTED** [review]

File: deploy/app.conf.yaml [hunk 5]
```diff
@@ -19,10 +19,14 @@ max_size_of_org_signature_pdfa: 204800
 
 cla_platform_url: "${CLA_PLATFORM_URL}"
 
+restricted_corp_email_suffix: ${RESTRICTED_CORP_EMAIL_SUFFIX}
```
Comment (zengchen1024) [inline deploy/app.conf.yaml:22]: how to define list of email suffix

**Review by zengchen1024 — APPROVED** [review]

File: deploy/app.conf.yaml (hunk 5 above)
Comment (xwz123) [reply deploy/app.conf.yaml:22]: use the list inline expression like: [qq.com,163.com]

**Pull request merged**